ops/createComponent.test.js

```diff
@@ -29,17 +29,32 @@ describe(`createComponent`, () => {
 
     it(`should throw an error`, async () => {
       try {
-        await createComponent({ path: `one`, name: `myComponent` })
+        await createComponent({ path: `app/one`, name: `myComponent` })
```

Comment:
`app` should be implicitly understood. As everything is always in `app`, I don't want to have to write `app/something` all the time. `something` is quite enough.

So I think we should change this test back to the way it was before. All commands are always about `app`.

However, if you are currently CWD to a directory that is not root and not in `app` folder, then it should fail.